src/main/kotlin/io/zerobase/smarttracing/resources/ModelsResource.kt

```diff
@@ -8,7 +8,7 @@ import javax.ws.rs.core.MediaType
 
 @Path("/models")
 @Produces(MediaType.APPLICATION_JSON)
-class ModelsResource(private val siteTypes: MultiMap<String, String>) {
+class ModelsResource(private val siteTypes: MultiMap<String, String>, private val scanTypes: List<String>) {
```

Comment:
the method is `getScannableTypes` but the field name is only `scanTypes`. rename this as `scannableTypes` for consistency